build/common/Makefile.common.mk

```diff
@@ -116,12 +116,12 @@ KIND_CLUSTER_NAME = kind-$(KIND_NAME)
 CONTROLLER_NAMESPACE ?= open-cluster-management-agent-addon
 KIND_VERSION ?= latest
 # Set the Kind version tag
-ifeq ($(KIND_VERSION), minimum)
-	KIND_ARGS = --image kindest/node:v1.19.16
-else ifneq ($(KIND_VERSION), latest)
-	KIND_ARGS = --image kindest/node:$(KIND_VERSION)
-else
-	KIND_ARGS =
+ifdef KIND_VERSION
```

Comment:
Why do we need this? we have default `latest`

Reply:
@yiraeChristineKim , not sure but this commit is just syncing the common Makefile in the `governance-policy-framework` repo so we can't change it here or else it'll be out of sync.